Automattic/woocommerce-payments
# Fix Task List Title Style

Fixes #5436 

#### Changes proposed in this Pull Request



* Fix how the task list title displays on the `Overview` page.
* Some CSS fixes to tidy up further.



#### Screenshots

**Before**

<img width="755" alt="Screenshot 2023-01-25 at 10 02 23" src="https://user-images.githubusercontent.com/11770181/214534738-58d322e5-47e7-4ced-b99b-82689f8d5818.png">

**After**

<img width="775" alt="Screenshot 2023-01-25 at 10 02 48" src="https://user-images.githubusercontent.com/11770181/214534809-f3b45a51-8ec4-4307-acb2-8090f45c4b13.png">






#### Testing instructions





* Go to `includes/class-wc-payments-account.php` and make the following change to the array returned by `get_account_status_data()`:

```php
		return [
			'email'               => $account['email'] ?? '',
			'country'             => $account['country'] ?? 'US',
			'status'              => 'restricted',
			'paymentsEnabled'     => $account['payments_enabled'],
			'deposits'            => $account['deposits'] ?? [],
			'depositsStatus'      => $account['deposits']['status'] ?? $account['deposits_status'] ?? '',
			'currentDeadline'     => $account['current_deadline'] ?? false,
			'pastDue'             => true,
			'accountLink'         => $this->get_login_url(),
			'hasSubmittedVatData' => $account['has_submitted_vat_data'] ?? false,
			'requirements'        => [
				'errors' => $account['requirements']['errors'] ?? [],
			],
		];
```

* Go to `Payments > Overview` and verify that the task list title matches the screenshot, that it is the same font size/style as the other headings on the page, and that everything looks as expected.

-------------------


**Post merge**
